thrust/detail/config/cpp_dialect.h

```diff
@@ -14,19 +14,66 @@
  *  limitations under the License.
  */
 
+/*! \file cpp_dialect.h
+ *  \brief Detect the version of the C++ standard used by the compiler.
+ */
+
 #pragma once
 
-#if   __cplusplus < 201103L
-  #define THRUST_CPP03
```

Comment:
I didn't carry over the `THRUST_CPPXX` macros, since they seem to be unused by the project. Let me know if I should add them back.

Reply:
The fewer macros the better, IMO.